modules/navier_stokes/src/fvkernels/NSFVMixturePhaseInterface.C

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NSFVMixturePhaseInterface.h"
+#include "NS.h"
+
+registerMooseObject("NavierStokesApp", NSFVMixturePhaseInterface);
+
+InputParameters
+NSFVMixturePhaseInterface::validParams()
+{
+  InputParameters params = FVElementalKernel::validParams();
+  params.addClassDescription("Implements a phase-to-phase volumetric exchange ");
+  params.addRequiredParam<MaterialPropertyName>(NS::alpha,
+                                                "Name of the volumetric exchange coefficient");
+  params.addRequiredParam<MooseFunctorName>("phase_coupled", "The ambient temperature");
+  return params;
+}
+
+NSFVMixturePhaseInterface::NSFVMixturePhaseInterface(const InputParameters & parameters)
+  : FVElementalKernel(parameters),
+    _alpha(getFunctor<ADReal>(NS::alpha)),
+    _phase_coupled(getFunctor<ADReal>("phase_coupled"))
+{
+}
+
+ADReal
+NSFVMixturePhaseInterface::computeQpResidual()
+{
+  auto elem_arg = makeElemArg(_current_elem);
+  const auto state = determineState();
+  return _alpha(elem_arg, state) * (_var(elem_arg, state) - _phase_coupled(elem_arg, state));
```

Comment:
The documentation claims that this adds to the residual of both phases, but this only adds to one of the residuals of `_var`. I'd suggest changing the documentation.

Reply:
Thanks. Updated the documentation. It now reads as follows:

```
!alert note
The kernel adds to the residual for the phase identified in `variable`.
The complementary kernel for the second phase is needed to preserve the phase fraction.
```